### ISSUE TITLE
SRS Review: Assumptions section

### ISSUE THREAD

**Issue by samm82** [balacij/Drasil#15]
# TODO
*Any point that requires more discussion should be pulled out to a different issue; I grouped these all here since they are related and small.*

- [x] There seems to be two assumptions in A1 (2D + observed at moment of load placement).
- [x] The definition of "slender" should be moved to the Terms and Defs section.
- [x] A2-A4 should be better formatted. Consider describing the beam as prismatic in the System Context section, then just listing A3 and A4 as their own assumptions (not as sub-assumptions of A2).
- [x] From [what I can tell](https://www.sciencedirect.com/topics/engineering/linear-elastic-model), "linear-elastic" shouldn't have a hyphen.
- [x] The wordings of A9 and A10 are confusing.
- [x] Either traceability between assumptions and what reference them is not included, or your assumptions are not used by the rest of the document.

**Comment by balacij** [comment]
Regarding A9 and A10, I think they are the now A15 and A16. I'm not quite sure what's confusing about it, can you elaborate please?

![Screenshot from 2023-03-16 16-41-58](https://user-images.githubusercontent.com/1627302/225747910-f49bf250-358b-4d96-bbdb-5e05b174f0d1.png)

**Comment by balacij** [comment]
Assumptions are underused, but necessary. I need Drasil's help to annotate it everywhere. I'll work on that with the Drasil code instead of the POC.

**Comment by samm82** [comment]
For A15 and A16, why not just say "The deflection function (y_B(x)) is continuously differentiable 4 times on [0,L]"? The current text seems too wordy.

**Comment by balacij** [comment]
Sure, that's fine too. I was initially thinking about $C^n$ (as in "the classes" in the standard sense) which is why it ended up like this.

**Comment by samm82** [comment]
Not sure I fully understand that notation but it's up to you ¯\\\_(ツ)_/¯

**Comment by balacij** [comment]
Oh, I changed it. Writing out the classes textually is definitely suboptimal, lol.
